hcl/strconv/quote.go

```diff
@@ -70,6 +71,9 @@ func Unquote(s string) (t string, err error) {
 					if q, _ := utf8.DecodeRuneInString(s); q == '"' {
 						continue
 					}
+					if q, _ := utf8.DecodeRuneInString(s); q == '\\' {
+						continue
+					}
```

Comment:
Fair - I was just so punch drunk that the copy paste seemed too hilarious not to do 😄